contributing-guides/style-guide.md

```diff
@@ -330,7 +330,16 @@ For instance, instead of `Listing all files:`, `List all files:` can be used as
  `ls`
 ```
 
-## Emphasis
+## Not rendered syntax
+
+Any syntax not documented on this guide or in the client specification is not guaranteed to be rendered in our clients. See common cases below.
+
+### Links and images
+
+- Do not use Markdown links (\[link text\](link url)) or images in the pages, they are not guaranteed to be supported by our clients.
```

Comment:
```suggestion
Any syntax not documented in this guide or the client specification will **not** be rendered in most of our clients. See common cases below.

### Links and images

- Do not use Markdown links (\[link text\](link url)) or images in the pages, they are not part of the tldr client specification and thus not supported by most of our clients.
```

@gutjuri's suggestions seems to be marked as outdated post merging with main, readded it here along with some minor changes.